src/sections/hero.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    
+  </head>
+  <body>
+    <section
+      class="overflow-hidden  ojemba-gradient1 font-railwayBold"
+    >
+      <nav class="flex justify-between text-xs text-white lg:text-xs">
+        <div class= "m-4 mt-8 sm:w-32 lg:h-16  w-16 lg:ml-20 md:mr-8 pt-3 ">
+          <img src="../../images/Logo.webp"     alt="" />
+        </div>
+        <div class="flex  lg:justify-between">
+          <img src="../../images/hamburger.webp" alt="" class="w-6 h-6 m-5 lg:hidden mt-8 mr-16">
+          <ul class="hidden lg:gap-8 mt-10 lg:flex  lg:justify-center cursor-pointer">
+            <li><a href="">Services</a></li>
+            <li><a href="">Über uns</a></li>
+            <li> <a href="">Darum wir</a></li>
+            <li> <a href="">Team</a></li>
+            <li><a href="">Über Ruanda</a> </li>
+            <li> <a href="">FAQ</a></li>
+            <li> <a href="">Kontakt</a></li>
+          </ul>
+          <div class="flex-col hidden mt-8 ml-4 mr-32 text-xs lg:flex"><span>DE</span> <span>EN</span></div>
+        </div>
+      </nav>
+      <div class="grid sm:grid-cols-2">
+        <div  class="pt-4 m-4 lg:ml-16 lg:mt-60 md:font-railwayBold">
+            <h1 class="text-sm text-white lg:text-5xl ">Win-Win: <br>
+                Europas <span class="text-green">Digitalisierung</span> <br>profitiert von Afrikas Talenten </h1>
+            <p class="mt-2 text-xs text-white lg:mt-6 font-railwayLight">Erweiteren Sie Ihre Entwicklungsteams mit Ojemba</p>
+        </div>                
+        <div class="overflow-x-hidden relative -bottom-20 "><img src="../../images/Ojembees.webp " ></div>  
+        
+      </div>
+      </section>
+
+      <footer class="ojemba-gradient1 text-white p-8 lg:px-44 flex flex-col gap-4 lg:flex-row lg:justify-between  ">
```

Comment:
I'm a bit confused about why you used the semantic HTML `<footer>` tag in the hero section. This might lead to confusion. It's generally recommended to use this tag when dealing with the footer of your page, rather than in other sections. It doesn't seem to make sense in this context.